.devcontainer/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM fedora:34
+
+# Install few packages for convenient usage.
+# Inspired by the devcontainer in osbuild.
+RUN dnf install -y \
+    fish \
+    fd-find \
+    ripgrep \
+    jq
+# We build composer using RPM, install the tooling
+RUN dnf install fedora-packager rpmdevtools go-srpm-macros -y
+# The list of requirements is specified in the spec file.
+# To install them, dnf needs the "builddep" command.
+RUN dnf install 'dnf-command(builddep)' -y
+# Copy the specfile, remove gometa because dnf fails
+# to parse it, and finally install the dependencies.
+COPY osbuild-composer.spec /tmp/osbuild-composer.spec
```

Comment:
This is also unnecessary now.

Reply:
Is it? Don't we want to install the dependencies for building RPM inside the container?

Reply:
I meant the copy command. :) You can just `dnf builddep osbuild-composer.spec`

Reply:
No, that is still needed for copying the file from the host to the container buildroot. Or am I missing something here?

Reply:
Right, this is Dockerfile, sorry, it's been so long since I wrote one. Of course, this is right. I read it like a bash script. :sweat_smile: